tests/testquic.nim

```diff
@@ -174,3 +174,36 @@ suite "Quic transport":
 
     # run multiple clients simultainiously
     await allFutures(runClient(), runClient(), runClient())
+
+    asyncTest "read/write Lp":
+      proc serverHandler(
+          server: QuicTransport
+      ) {.async: (raises: [transport.TransportError, LPStreamError, CancelledError]).} =
+        while true:
+          let conn =
+            try:
+              await server.accept()
+            except QuicTransportAcceptStopped:
+              return # Transport is stopped
+          if conn == nil:
+            continue
+
+          let stream = await getStream(QuicSession(conn), Direction.In)
+          check (await stream.readLp(100)) == fromHex("1234")
+          await stream.writeLp(fromHex("5678"))
+          await stream.close()
+
+      proc runClient(server: QuicTransport) {.async.} =
+        let client = await createTransport()
+        let conn = await client.dial("", server.addrs[0])
+        let stream = await getStream(QuicSession(conn), Direction.Out)
+        await stream.writeLp(fromHex("1234"))
+        check (await stream.readLp(100)) == fromHex("5678")
+        await client.stop()
```

Comment:
Nits:
1. unless you're planning to use this proc on other tests, i'd say unpack it (not have it).
2. `await client.stop()` could be `defer: await client.stop()` right after creating the transport, no?